homework/01LessonOne.js

```diff
@@ -106,7 +108,8 @@ var notNotTrue;
 // What is the difference between the "and" and the "or" operators? Explain why you would use each of them.
 //
 // Your Answer Goes Here:
-//
+// "and" returns the value on the left while "or" returns the one of the right. You would use each depending on which variable you need to compare. 
```

Comment:
The 'and' operator evaluates the `truthiness` of the values on both values and only returns true if BOTH (left and right) are true.

The 'or' operator evaluates the `truthiness` of the values on both values and returns true if EITHER (left and right) are true.